.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: python
 python:
- - 2.7
+ - 3.6
```

Comment:
For some reason, Travis fails and says it can't find the Python 3.6 interpreter when we do it on 2.7. The 2.7 tests seem to work fine when we tell Travis to use 3.6. Apparently just some quirk with the Travis and `tox` interactions.